packages/react-components/react-combobox/src/utils/useComboboxPopup.ts

```diff
@@ -0,0 +1,38 @@
+import { resolvePositioningShorthand, usePositioning } from '@fluentui/react-positioning';
+import { ExtractSlotProps, Slot, useMergedRefs } from '@fluentui/react-utilities';
+import type { ComboboxBaseProps } from './ComboboxBase.types';
+import { Listbox } from '../components/Listbox/Listbox';
+
+export function useComboboxPopup(
+  props: ComboboxBaseProps,
+  triggerShorthand?: ExtractSlotProps<Slot<'button'>>,
+  listboxShorthand?: ExtractSlotProps<Slot<typeof Listbox>>,
```

Comment:
nit: We use term "shorthand" to unresolved slot props i.e. a union of `string | number | SlotProps`. Is there sense to rename it to `triggerSlotProps` & `listboxSlotProps` to make it clearer? (it will be a clear indicator that a shorthand was resolved to props already)